README.md

```diff
@@ -1,20 +1,21 @@
-# Membrane Multimedia Framework: FFmpeg H264 Element
+# Membrane H264 FFmpeg plugin
 
-[![CircleCI](https://circleci.com/gh/membraneframework/membrane-element-ffmpeg-h264.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane-element-ffmpeg-h264)
+[![CircleCI](https://circleci.com/gh/membraneframework/membrane_h264_ffmpeg_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_h264_ffmpeg_plugin)
```

Comment:
```suggestion
[![Hex.pm](https://img.shields.io/hexpm/v/membrane_ffmpeg_h264_plugin.svg)](https://hex.pm/packages/membrane_ffmpeg_h264_plugin)
[![API Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_ffmpeg_h264_plugin/)
[![CircleCI](https://circleci.com/gh/membraneframework/membrane_h264_ffmpeg_plugin.svg?style=svg)](https://circleci.com/gh/membraneframework/membrane_h264_ffmpeg_plugin)
```

I don't know if we want each repo contains all badges but if yes you can add them.